### PR TITLE
Robot mapping

### PR DESCRIPTION
**Pull request by ryanlarkin**
Not fully documented yet, but this is the code used in drive practice

### CODE REVIEW

**Comment by cnnradams** [comment]
No its not ready yet wait till tomorrow

**Comment by cnnradams** [comment]
We dont merge undocumented and unformatted code into master

**Comment by kevlam2706** [comment]
Controls team is meeting tonight to work on installing the bling and the ring light. You guys are welcome to come and work on software during that time. Or you can do this from home.

**Comment by ryanlarkin** [comment]
Do you want me to work on documenting this?

**Comment by kevlam2706** [comment]
Let's work on getting all of our code cleaned up, documented, and in master so we don't waste time trying to keep track of which code is in who's branch, especially code that the drive team needs.

-k

File: src/org/usfirst/frc/team2706/robot/commands/mechanismcontrol/CloseGearMechanism.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.usfirst.frc.team2706.robot.commands.mechanismcontrol;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class CloseGearMechanism extends Command {
```
Comment (cnnradams) [inline src/org/usfirst/frc/team2706/robot/commands/mechanismcontrol/CloseGearMechanism.java]: documentation?

File: src/org/usfirst/frc/team2706/robot/commands/teleop/TakeOverCommandBrake.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package org.usfirst.frc.team2706.robot.commands.teleop;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * Until interrupted, stops robot from driving around, and goes into brake mode to stop the robot
+ */
+public class TakeOverCommandBrake extends Command {
```
Comment (cnnradams) [inline src/org/usfirst/frc/team2706/robot/commands/teleop/TakeOverCommandBrake.java]: delete that

**Review by cnnradams — CHANGES_REQUESTED** [review]
fix small thing

**Review by cnnradams — APPROVED** [review]
because were rushed

**Pull request merged**